packages/core/src/baseResourceApi.ts

```diff
@@ -94,27 +94,29 @@ export abstract class BaseResourceAPI<ResponseType> {
    */
   protected pickDateProps<T = ResponseType>(
     parents: string[],
-    props: NonNullable<KeysOfType<NoInfer<T>, Date | undefined>>[]
+    props: NonNullable<KeysOfType<NoInfer<T>, Date | undefined>>[],
   ): [string[], string[]] {
     return [parents, props as string[]];
   }
 
-  protected url(path: string = '') {
-    return this.resourcePath + '/' + path;
+  protected url(path = '') {
+    return `${this.resourcePath}/${path}`;
   }
 
   private requestWrapper(request: HttpCall) {
     return async <ResponseType>(
       path: string,
-      options?: HttpRequestOptions
+      options?: HttpRequestOptions,
     ): Promise<HttpResponse<ResponseType>> => {
-      if (options !== undefined)
-        options = {
-          ...options,
-          params: DateParser.parseFromDates(options.params),
-          data: DateParser.parseFromDates(options.data),
-        };
-      const response = await request<ResponseType>(path, options);
+      const requestOptions =
```

Comment:
Had to define a new variable as before we reused the argument variable name (linter complained)